recipes/dtl/all/conanfile.py

```diff
@@ -18,7 +17,6 @@ class DtlConan(ConanFile):
 
     package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
-    no_copy_source = True
```

Comment:
@toge we can keep this if we move the replace_in_file to the source() method, as every configuration will need it, and we save some space :)

Reply:
Thank you for your review!
I agree with you. Fixed.